exercises/accumulate/examples/src/Accumulate.purs

```diff
@@ -0,0 +1,14 @@
+module Accumulate
+  ( accumulate
+  ) where
+
+import Data.List (List(Nil), (:), reverse)
+
+
+accumulate :: forall a b . (a -> b) -> List a -> List b
+accumulate f values = accumulate' f Nil values
+
+
+accumulate' :: forall a b . (a -> b) -> List b -> List a -> List b
```

Comment:
I don't have a strong preference, but sometimes `let` can be useful if you want to place a let between two function arguments, which you can't do with `where`.